tests/helpers.js

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {IdEncoder, IdGenerator} from 'bnid';
+
+export const generateMultibase = async ({bitLength = 32}) => {
+
+  const generator = new IdGenerator({bitLength});
+
+  // base58, multibase, fixed-length encoder
+  const encoder = new IdEncoder({
+    encoding: 'base58',
+    fixedLength: true,
+    multibase: true
+  });
+  return encoder.encode(await generator.generate());
+};
+
+/*
+ * Using a colon (:) as the delimiter, split the identifier
+ * into its components: a scheme, a method, a version, and a
+ * multibaseValue. If there are only three components set the
+ * version to the string value 1 and use the last value as
+ * the multibaseValue.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} did - A did as a string.
```

Comment:
Do we no longer need to write this as `options.did`?

Reply:
this is actually a really good catch and a really good question. I also found that a jsdoc line that ended with a `;` instead of a `.` that the linter was also not throwing on so I'll review the jsdoc stuff: https://github.com/w3c-ccg/did-key-test-suite/pull/19/commits/a2facd2a22917d134774a9caabc553e39a99aede

Also thanks for the README reminder.